trade-finance-manager-api/src/v1/controllers/utilisation-reports/fee-record-correction/delete-fee-record-correction-transient-form-data.controller.ts

```diff
@@ -0,0 +1,32 @@
+import { isAxiosError, HttpStatusCode } from 'axios';
+import { Response } from 'express';
+import api from '../../../api';
+import { CustomExpressRequest } from '../../../../types/custom-express-request';
+
+export type DeleteFeeRecordCorrectionTransientFormDataRequest = CustomExpressRequest<{
+  params: {
+    reportId: string;
+    feeRecordId: string;
+    user: string;
+  };
+}>;
+
+/**
+ * Deletes the users fee record correction transient form data for the given fee record id
+ * @param req - The request object
+ * @param res - The response object
+ */
+export const deleteFeeRecordCorrectionTransientFormData = async (req: DeleteFeeRecordCorrectionTransientFormDataRequest, res: Response) => {
+  try {
+    const { reportId, feeRecordId, user } = req.params;
+
+    await api.deleteFeeRecordCorrectionTransientFormData(reportId, feeRecordId, user);
+
+    return res.sendStatus(HttpStatusCode.NoContent);
+  } catch (error) {
+    const errorMessage = 'Failed to delete fee record correction transient form data';
+    const errorStatus = (isAxiosError(error) && error.response?.status) || HttpStatusCode.InternalServerError;
```

Comment:
this is a little confusing to follow, documentation for what it does?

Reply:
Added some docs to the `@returns` tag in the TSDoc, I believe this is repeated for all endpoints but hopefully this should make it a bit clearer